src/Microsoft.Extensions.Logging.Log4Net.AspNetCore/Log4NetParams.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Microsoft.Extensions.Logging
+{
+    public class Log4NetParams
+    {
+        public string FileName = null;
```

Comment:
Per convention, this class should expose properties instead fields. It is okay for me that both properties have default values, but to be consistent, I prefer that you can change it member types.